tests/unit/test_prompt.py

```diff
@@ -1,13 +1,12 @@
-from unittest import TestCase, mock
+from unittest import mock
 
 from dvc.prompt import confirm
 
 
-class TestConfirm(TestCase):
-    @mock.patch("sys.stdout.isatty", return_value=True)
-    @mock.patch("dvc.prompt.input", side_effect=EOFError)
-    def test_eof(self, mock_input, mock_isatty):
-        ret = confirm("message")
-        mock_isatty.assert_called()
-        mock_input.assert_called()
-        self.assertFalse(ret)
+def test_confirm_in_tty_if_stdin_is_closed():
+    with mock.patch("dvc.prompt.input", side_effect=EOFError) as mock_input:
+        with mock.patch("sys.stdout.isatty", return_value=True) as mock_isatty:
```

Comment:
Let's use `mocker` fixture instead.

Reply:
BTW it seems there are quite a few `unittest.mock` usages inside pytest tests. For example in https://github.com/iterative/dvc/blob/d8b67f6f78781154aab39783401c181089468b02/tests/func/test_analytics.py#L9-L14

Should all `unittest.mock` usages be eliminated as a part of #1819?